### ISSUE TITLE
Spanish docs - Links don't work

### ISSUE THREAD

**Issue by omarespejel** [huggingface/transformers#17461]
## TLDR
Links to docs that don't exist (eg ./main_classes/pipelines) don't work. Issue first mentioned in #17349.

## Description
Currently, the links lead to an error if the doc is not yet translated. For example, this fragment in [`autoclass_tutorial`](https://huggingface.co/docs/transformers/main/es/autoclass_tutorial) leads a to an error for not having [model_doc/auto.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/auto.mdx)
 translated yet.
> Finalmente, las clases AutoModelFor te permiten cargar un modelo preentrenado para una tarea dada (revisa [aquí](https://huggingface.co/docs/transformers/main/es/model_doc/auto) para conocer la lista completa de tareas disponibles).

## Possible solution
Linking to the English docs while the Spanish versions become available.

## Possible reviewers
@sgugger @mishig25

**Comment by mishig25** [comment]
@omarespejel thanks a lot for notifying. I'm working on a fix
